src/app/store.ts

```diff
@@ -0,0 +1,13 @@
+import { createStore, combineReducers } from "redux";
```

Comment:
Instead of adding new and deleting `store.js` I think we should have renamed them so that github can keep track of changes.

Reply:
You're right, but there was almost nothing in the store :)